docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -403,42 +413,42 @@ In the InfluxDB data source, the legacy variable `$interval` is the same variabl
 
 The InfluxDB and Elasticsearch data sources have `Group by time interval` fields that are used to hard code the interval or to set the minimum limit for the `$__interval` variable (by using the `>` syntax -> `>10m`).
 
-### $\_\_interval_ms
+### `$__interval_ms`
 
 This variable is the `$__interval` variable in milliseconds, not a time interval formatted string. For example, if the `$__interval` is `20m` then the `$__interval_ms` is `1200000`.
 
-### $\_\_name
+### `$__name`
 
-This variable is only available in the Singlestat panel and can be used in the prefix or suffix fields on the Options tab. The variable is replaced with the series name or alias.
+This variable is only available in the **Singlestat** panel and can be used in the prefix or suffix fields on the Options tab. The variable is replaced with the series name or alias.
 
-{{% admonition type="note" %}}
-The Singlestat panel is no longer available from Grafana 8.0.
-{{% /admonition %}}
+{{< admonition type="note" >}}
+The **Singlestat** panel is no longer available from Grafana 8.0.
+{{< /admonition >}}
```

Comment:
Not for this PR but this can probably be removed?

Reply:
There's another part of the doc that references this panel, so that's why I've left it in.